configure.py

```diff
@@ -1116,7 +1111,7 @@ def configure_node(o):
   o['variables']['node_no_browser_globals'] = b(options.no_browser_globals)
   # TODO(refack): fix this when implementing embedded code-cache when cross-compiling.
   if o['variables']['want_separate_host_toolset'] == 0:
-    o['variables']['node_code_cache_path'] = 'yes'
+    o['variables']['node_code_cache'] = 'yes'
```

Comment:
```suggestion
    o['variables']['node_code_cache'] = 'yes'  # For testing
```

Reply:
@joyeecheung Thanks, I've added the comment now.